demos/classification_demo/python/classification_demo.py

```diff
@@ -37,73 +38,95 @@
 log.basicConfig(format='[ %(levelname)s ] %(message)s', level=log.DEBUG, stream=sys.stdout)
 
 
-def build_argparser():
+def parse():
+    def print_version():
+        log.info('OpenVINO Runtime')
+        print('\tbuild {}'.format(get_version()))
+
+    def print_key_bindings():
+        print('\n\tKey bindings:\n\t\tQ, q - Quit\n\t\tP, p, 0, SpaceBar - Pause')
```

Comment:
Space is not pause if the input is a folder

Reply:
No, it still a key binding. It just does not what you claim

Reply:
Instead of
> \tP, p, 0, SpaceBar - Pause for camera id/Switch frame for separated images

it should say

> \tP, p, 0, SpaceBar - Pause for stream input. Any key - switch frame for separated images